BFAIR/INCA/INCA_reimport.py

```diff
@@ -0,0 +1,996 @@
+# -*- coding: utf-8 -*-
+"""Data re-import module."""
+
+import time
+
+# System
+from math import isnan, isinf
+import re
+
+# Dependencies from 3rd party
+from molmass.molmass import Formula
+import scipy.io
+import os
+from datetime import datetime
+from stat import ST_SIZE, ST_MTIME
+
+__version__ = "0.0.1"
+
+
+class INCA_reimport:
+    def __init__(self):
+        self.fittedData = []
```

Comment:
Please note that when I wrote this Pandas did not exist.  I would recommend relying on Pandas for the final outputs or at least using data structures that fit well into the Pandas toolchain.

Reply:
Will do